tools/src/vendoring/config/expoGoConfig.ts

```diff
@@ -58,6 +58,7 @@ const config: VendoringTargetConfig = {
       source: 'https://github.com/software-mansion/react-native-gesture-handler.git',
       semverPrefix: '~',
       ios: {},
+      android: {},
```

Comment:
would you like to remove the vendoring code in _legacy.ts_ ?

Reply:
I planned to do that, but there are more libs to delete from there, so I'll do that separately